openpype/tools/sceneinventory/window.py

```diff
@@ -139,6 +139,11 @@ def keyPressEvent(self, event):
 
         """
 
+    def _refresh_callback(self):
```

Comment:
Can we rename this `_on_refresh` just so it aligns with the other callback `_on_update_all` and I believe also aligns more with how callbacks are more often set up in OpenPype in tools?

Reply:
I wanted, but I would expect that 'on refresh' would be triggered during/before/after refresh, not to trigger refresh.

Reply:
Maybe `_on_refresh_request`?

Reply:
Yup, sounds like a good alternative to me.